sdk/servicebus/Azure.Messaging.ServiceBus/tests/ManagementClient/ServiceBusManagementClientLiveTests.cs

```diff
@@ -113,16 +113,16 @@ public async Task BasicTopicCrudOperations()
             TopicDescription updatedTopic = await client.UpdateTopicAsync(getTopic);
             Assert.AreEqual(getTopic, updatedTopic);
 
-            var exists = await client.TopicExistsAsync(topicName);
+            bool exists = await client.TopicExistsAsync(topicName);
             Assert.True(exists);
 
             List<TopicDescription> topicList = new List<TopicDescription>();
             await foreach (TopicDescription topic in client.GetTopicsAsync())
             {
                 topicList.Add(topic);
             }
-            Assert.True(topicList.Count == 1);
-            Assert.AreEqual(topicList.First().Name, topicName);
+            Assert.True(topicList.Count >= 1);
```

Comment:
Why do we need to weaken the assertion?

Reply:
If we run all the tests locally using same namespace then one test can effect other one. Let's say in this test if we assert `Assert.True(topicList.Count == 1);` then it may fail because other test might have created topics and it could effect the topic count.

I know for nightly run we different namespace for each test but I thought it would be better if someone uses same namespace locally.

Reply:
Hmm, that is annoying for running locally, but I would rather not change this as we could be hiding failures.

> > Can we get rid of the Filters namespace and just move under management?
> 
> I thought about moving filters to the management but in future if we add the rules in amqp then again we would have to move filters in here. I don't have strong opinion on this one, I'm fine either way.

That's true. I don't think it would be the end of the world though if we added rules back in amqp but the filters were under management namespace.

Reply:
@jsquire thoughts about the test issue?

Reply:
We can use [[NonParallelizable]](https://github.com/nunit/docs/wiki/NonParallelizable-Attribute)?